### ISSUE TITLE
ENH add a button to restart a failed submission

### ISSUE THREAD

**Issue by tomMoral** [paris-saclay-cds/ramp-board#482]
On the server, some submission fails for the wrong reason (issue in the dispatcher, timeout in AWS workers, missing library).
We can restart them directly on the server but this is not really a good solution (we should reduce the amount of time we `ssh` the server ).
What we could do it add a button in the admin dashboard, for failed submission to reset there state to `new` so they can be retrained.

**Comment by rth** [comment]
yes, this would be very useful and shouldn't take too much work.
